src/ion/data_structures.rs

```diff
@@ -336,10 +336,6 @@ pub struct Env<'a, F: Function> {
     pub liveouts: Vec<IndexSet>,
     pub blockparam_outs: Vec<BlockparamOut>,
     pub blockparam_ins: Vec<BlockparamIn>,
-    /// Blockparam allocs: block, idx, vreg, alloc. Info to describe
-    /// blockparam locations at block entry, for metadata purposes
-    /// (e.g. for the checker).
-    pub blockparam_allocs: Vec<(Block, u32, VRegIndex, Allocation)>,
```

Comment:
I also did this driveby removal and a few under `cfg(feature = "checker")` below as I noticed I hadn't fully removed the machinery that the old checker design required.